### ISSUE TITLE
[BUG] can't set the "bottom" and "top" forms for danmaku drawing

### ISSUE THREAD

**Issue by iyear** [DIYgod/DPlayer#1233]
I found that using `dp.danmaku.draw()` in the live broadcast mode can only draw danmaku in the form of `right`. I checked the source code and found that the code and the document did not match.

I checked the previous issue and found #947, but he did not completely solve the problem

code:
https://github.com/DIYgod/DPlayer/blob/808dd623d6b3facf321f27f9358c5fe97b3931a3/src/js/danmaku.js#L207

After a conversion, the number is converted to a string

doc:
https://github.com/DIYgod/DPlayer/blob/master/docs/zh/guide.md#api

Directly pass in `right` `bottom` `top` instead of numbers

In other words, `utils.number2Type` always passes through the `default` branch, which is `right`, which should be the problem

Just delete the line https://github.com/DIYgod/DPlayer/blob/808dd623d6b3facf321f27f9358c5fe97b3931a3/src/js/danmaku.js#L207
and the no longer used `number2Type`

I can submit a simple pr to solve this problem
